### DISCUSSION TITLE
Cancel example?

### DISCUSSION THREAD

**Discussion by ehp246** [ical4j/ical4j#777] [Q&A]
I'm having a hard time to get cancellation working. The following are the original invite and the cancel ics. I tried to load these two files in the order to Google and Outlook. I got two meetings created for the two ICS. I was hoping after the second ICS is loaded, the original meeting will be cancelled.

The invite:
```
BEGIN:VCALENDAR
PRODID:-//Events Calendar//iCal4j 1.0//EN
CALSCALE:GREGORIAN
VERSION:2.0
BEGIN:VEVENT
DTSTAMP:20200805T194909Z
DTSTART:20200805T194909Z
DTEND:20200805T200409Z
SUMMARY:Telehealth visit
UID:82aaf1aa-9522-4000-8a29-084c5a4762d5
METHOD:REQUEST
ATTENDEE;ROLE=REQ-PARTICIPANT;CN=Lei Yang:mailto:lei.yang@bigr.io
ORGANIZER;ROLE=REQ-PARTICIPANT;CN=Lei Yang:mailto:lei.yang@amwell.com
URL:https://www.yahoo.com
LOCATION:https://www.yahoo.com
END:VEVENT
END:VCALENDAR

```

The cancellation:
```
BEGIN:VCALENDAR
PRODID:-//Events Calendar//iCal4j 1.0//EN
CALSCALE:GREGORIAN
VERSION:2.0
BEGIN:VEVENT
DTSTAMP:20200805T194909Z
DTSTART:20200805T194909Z
DTEND:20200805T200409Z
SUMMARY:Telehealth visit
UID:82aaf1aa-9522-4000-8a29-084c5a4762d5
METHOD:CANCEL
ATTENDEE;ROLE=REQ-PARTICIPANT;CN=Lei Yang:mailto:lei.yang@bigr.io
ORGANIZER;ROLE=REQ-PARTICIPANT;CN=Lei Yang:mailto:lei.yang@amwell.com
URL:https://www.yahoo.com
LOCATION:https://www.yahoo.com
END:VEVENT
END:VCALENDAR
```

**Comment by ehp246** [comment]
By the way, I'm running on v4 alpha 6.

**Comment by benfortuna** [comment]
The iTIP RFC might help with this:

https://tools.ietf.org/html/rfc5546#section-3.2.5

From what I can see the only missing property the specification suggests is mandatory is SEQUENCE. This is what indicates the order of the iCalendar objects. I think if you just add a "SEQUENCE:2" that might be enough..

ps. Thanks for feedback on the 4.0 alpha builds.
